src/foam/comics/DAOControllerView.js

```diff
@@ -149,14 +149,17 @@ foam.CLASS({
             .callIfElse(this.data.primaryAction, function() {
               this.startContext({ data: self })
                 .start()
-                  .add(self.data.primaryAction)
+                  .tag(self.data.primaryAction, { size: 'LARGE' })
```

Comment:
Are we getting rid of 'size'? Is this PR still valid?

Reply:
No, size is just moving from the Action itself to ActionView.